README.md

```diff
@@ -1,10 +1,11 @@
 # Hibernate ORM 5 as Java 9 automatic modules
 
-A simple test bed for using Hibernate ORM 5 as (automatic) JPMS modules.
+A simple test bed for using Hibernate ORM 5.3 as (automatic) JPMS modules.
+
 Some things noteworthy:
 
 * Almost all dependencies are used as automatic modules, with the exception of `java.transaction` and `java.persistence`;
 * The JDK's `java.transaction` module is upgraded with _org.jboss.spec.javax.transaction:jboss-transaction-api\_1.2\_spec:2.0.0.Alpha1_, which adds all JTA types (with the exception of the XA package conflicting with the _java.sql_ module)
-* _hibernate-jpa-2.1-api-1.0.0.Final.jar_ can't be used as an automatic module as the naming algorithm stumples upon the "2.1" within the JAR name; so instead ModiTect (https://github.com/moditect/moditect/) is used to convert the JPA API JAR into an explicit JPMS module
+* _javax.persistence-api-2.2.jar_ can't be used as an automatic module as the it needs a `uses javax.persistence.spi.PersistenceProvider` clause in its module definition; so instead ModiTect (https://github.com/moditect/moditect/) is used to convert the JPA API JAR into an explicit JPMS module and we add the missing directive.
```

Comment:
That's weird, as an automatic module I thought it should be able to _use_ all services provided.

Btw. also check out the "orm_modularized" branch which doesn't use automatic modules but uses ModiTect to create named modules for ORM, too.

Reply:
Feel free to play with it. Without the explicity import it didn't work for me, but maybe I misunderstood the reason.